actionpack/test/dispatch/request/json_params_parsing_test.rb

```diff
@@ -37,9 +37,9 @@ def teardown
     )
   end
 
-  test "parses json params for application/vnd.api+json" do
+  test "does not parses unregistered media types such as application/vnd.api+json" do
     assert_parses(
-      {"person" => {"name" => "David"}},
+      {},
```

Comment:
ha, that was quick!  you merged it right before I revised the commit message from 'I' to 'we' and fixed a typo.  :) Thanks!